services/storage/src/simcore_service_storage/redis.py

```diff
@@ -0,0 +1,35 @@
+import logging
+from typing import cast
+
+from aiohttp import web
+from servicelib.redis import RedisClientSDK
+from settings_library.redis import RedisDatabase, RedisSettings
+
+from .constants import APP_CONFIG_KEY
+from .settings import Settings
+
+_logger = logging.getLogger(__name__)
+
+_APP_REDIS_KEY = "APP_REDIS_KEY"
+
+
+def setup_redis(app: web.Application):
+    async def _setup(app: web.Application):
+        app[_APP_REDIS_KEY] = None
+        settings: Settings = app[APP_CONFIG_KEY]
+        assert settings.STORAGE_REDIS  # nosec
+        redis_settings: RedisSettings = settings.STORAGE_REDIS
+        redis_locks_dsn = redis_settings.build_redis_dsn(RedisDatabase.LOCKS)
+        app[_APP_REDIS_KEY] = client = RedisClientSDK(redis_locks_dsn)
+        await client.setup()
+
+        yield
+
+        if client:
+            await client.shutdown()
```

Comment:
Let's use this pattern please
```suggestion
        app[_APP_REDIS_KEY] = None
        settings: Settings = app[APP_CONFIG_KEY]
        assert settings.STORAGE_REDIS  # nosec
        redis_settings: RedisSettings = settings.STORAGE_REDIS

        app[_APP_REDIS_CLIENTS_MANAGER] = manager = RedisClientsManager(
            databases={
                RedisDatabase.LOCKS,
            },
            settings=redis_settings,
        )
    
        await manager.setup()
    
        yield
    
        await manager.shutdown()
```

Reply:
Cool thanks, I am writing a note down and will refactor also with other Redis clients with this pattern (for example in the Resource usage tracker) in different PR. Some tests started to fail after the change so I will need to spend more time on it and I already moved to something else.